crowbar_framework/app/views/repositories/index.html.haml

```diff
@@ -0,0 +1,80 @@
+.col-xs-12
+  .row
+    %h1.page-header
+      = t(".title")
+    .alert.alert-info
+      = t(".refresh_hint")
+
+  - repository_platforms.each_with_index do |platform, index|
+    .row
```

Comment:
If I look at other pages we are not wrapping the panel in another `.row` tag, e.g. https://github.com/MaximilianMeister/crowbar-core/blob/master/crowbar_framework/app/views/barclamp/index.html.haml#L11